remix.init/README-edge.md

```diff
@@ -40,26 +40,26 @@ npm install
 Run
 
 ```sh
-netlify dev
+npm run dev
```

Comment:
We use the new v2 dev mode, which allows remiox dev to run a custom server, which in our case is `ntl dev`. For that reason we dont want them running it themselves